### DISCUSSION TITLE
Noob, how to change the -e EULA=TRUE

### DISCUSSION THREAD

**Discussion by WaLanio** [itzg/docker-minecraft-server#1169] [Q&A] [answered]
If i start the container, it stops directly before i can in the therminal of the container to say -e EULA=TRUE.
The time he is running like 2a3 seconds.
Is it true that i have to do this so?

**Comment by itzg** [comment]
Yes that is required but you don't enter it in the container console. It's an environment variable setting when creating the container.

**Reply by WaLanio** [reply]
where do i have to do it then?

**Reply by itzg** [reply]
How are you creating your container? I need more information to help you.

**Comment by royvandyck** [comment, marked as answer]
Hey! so I'm having the same problems. I'm trying to run your image on my Synology nas DS920+ . And i'm doing it via the docker package. So it automaticaly runs the startup command where i have to put in the EULA thing. and i can't seem to find a way to change it.
i can change the enviroment settings but thats all i can see that would really make a difference (this is the place where i changed amount of RAM, Type, Version) but its more of a fill in the information in the right square layout.

**Reply by royvandyck** [reply]
Found out how to do it

Just add a variable yourself and make it EULA and TRUE in the enviroment section

**Reply by WaLanio** [reply]
A oke, I will try that way!
